gnucash/gnome/dialog-price-edit-db.cpp

```diff
@@ -88,14 +89,14 @@ typedef struct
 
     GtkWidget *remove_dialog;
     GtkTreeView *remove_view;
-    gint remove_source;
+    int remove_source;
 } PricesDialog;
```

Comment:
To convert straight away to
```
struct PricesDialog {};
```
to avoid the typedef or will that not work here ?

I know it's not directly related to the topic of the PR. Just suggesting as you have made several other changes to convert the file to C++.